tests/unit/source_code/notebooks/01_HL7Streaming.scala

```diff
@@ -0,0 +1,221 @@
+// Databricks notebook source
```

Comment:
let's be consistent and add `.sample` suffix also to non-python notebooks. 

you can also add `.txt` :)

Reply:
done